<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSensorDisplayClock</class>
 <widget class="QMainWindow" name="FrmSensorDisplayClock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(2,87, 122);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView_backgroundClock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>721</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Dải đo</string>
    </property>
    <widget class="QSpinBox" name="spinBox_rangeMin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
}
QSpinBox:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>-999999</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_rangeMax">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
}
QSpinBox:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>-999999</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Từ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Đến</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Đơn vị đo</string>
    </property>
    <widget class="QComboBox" name="comboBox_unit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
}
QComboBox::drop-down{
width: 20;
}
QComboBox:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Độ ẩm tương đối (%RH)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Độ ẩm tuyệt đối (g/m3)</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>671</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cảm biến</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Thông tin thiết bị</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(250, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 6px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
background-color: rgb(25, 149, 190);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../device.qrc">
      <normaloff>:/icon/device/iconDeviceOff_401.jpg</normaloff>
      <activeoff>:/icon/device/iconDeviceOn_401.jpg</activeoff>:/icon/device/iconDeviceOff_401.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Thông tin thiết bị</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(3, 87, 123);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 8px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../group.qrc">
      <normaloff>:/icon/group/iconSetup_On1.jpg</normaloff>
      <activeoff>:/icon/group/iconSetup_On.jpg</activeoff>:/icon/group/iconSetup_On1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>527</y>
      <width>81</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>graphicsView_backgroundClock</tabstop>
  <tabstop>comboBox_unit</tabstop>
  <tabstop>spinBox_rangeMin</tabstop>
  <tabstop>spinBox_rangeMax</tabstop>
 </tabstops>
 <resources>
  <include location="../device.qrc"/>
  <include location="../group.qrc"/>
 </resources>
 <connections/>
</ui>
